pkg/handlers/webhookreceiver_test.go

```diff
@@ -73,18 +72,13 @@ var _ = Describe("Webhook Handlers", func() {
 		}
 		testAlert = testconst.NewTestAlert(false, false)
 		testAlertResolved = testconst.NewTestAlert(true, false)
-		activeServiceLog = NewTestServiceLog(
-			ServiceLogActivePrefix+": "+testconst.ServiceLogSummary,
-			testconst.ServiceLogActiveDesc+"\nReferences: [\""+slRefs+"\"]",
-			"",
-			testconst.TestNotification.Severity,
-			testconst.TestNotification.LogType)
 		resolvedServiceLog = NewTestServiceLog(
```

Comment:
Seems we build the ServiceLog but we never checked the values inside?

Reply:
Yeah, I might be ~too lazy~ to replace the `activeServiceLog` instance to use `gomock.Any()` in mocking to less strictly validating the specifics of the `ServiceLog` being passed but benefits of flexibility.  

I agree, use the actual activeServiceLog might be better, let me do some updates accordingly. 
cc: @hectorakemp - I think it also related  to your following questions regarding this building ServiceLog instance?